app/features/receive/scan.tsx

```diff
@@ -0,0 +1,47 @@
+import { getDecodedToken } from '@cashu/cashu-ts';
+import {
+  PageBackButton,
+  PageContent,
+  PageHeader,
+  PageHeaderTitle,
+} from '~/components/page';
+import { QRScanner } from '~/components/qr-scanner';
+import { useToast } from '~/hooks/use-toast';
+import { useNavigateWithViewTransition } from '~/lib/transitions';
+
+export default function Scan() {
+  const navigate = useNavigateWithViewTransition();
+  const { toast } = useToast();
+
+  return (
+    <>
+      <PageHeader>
+        <PageBackButton
+          to="/receive"
+          transition="slideDown"
+          applyTo="oldView"
+        />
+        <PageHeaderTitle>Scan</PageHeaderTitle>
+      </PageHeader>
+      <PageContent className="items-center justify-center">
+        <QRScanner
```

Comment:
this is how scan looks for me on desktop:

![Screenshot 2025-01-23 at 22 23 26](https://github.com/user-attachments/assets/a3b9ded5-e68f-4b9b-a012-baaaaa473eb0)